common/src/main/java/io/netty/util/NetUtil.java

```diff
@@ -264,7 +268,22 @@ public Integer run() {
                             logger.debug("{}: {}", file, somaxconn);
                         }
                     } else {
-                        if (logger.isDebugEnabled()) {
+                        // Try to get from sysctl
+                        Integer tmp = null;
+                        String sysctlCmd = SystemPropertyUtil.get("io.netty.net.sysctl", "sysctl");
```

Comment:
@normanmaurer - FYI. This is to get the max con on systems that don't have `/proc` by default (e.g. MacOS). I can set this property to default if we don't want to start-up a new process during initialization ... not sure if this will be too heavy weight.